### ISSUE TITLE
__DBT_TMP tables left undeleted after built with incremental models

### ISSUE THREAD

**Issue by djamadeus** [tglunde/dbt-exasol#85]
when running incremental models like the following, the __DBT_TMP tables are left undeleted in the schema.

```
{{
    config(
        materialized='incremental',
        unique_key = ['matsid', 'crdate']
    )
}}

Select
 "matsid" as matsid,
 "crdate" as crdate,
 "min_lager" as min_lager,
 "max_lager" as max_lager,
 "differenz" as differenz,
 "active" as active
FROM {{ source('VIRTUAL_JOINTDB_HAUPTLAGER','"bestand_aktiv_liste"') }}

{% if is_incremental() %}

    {%- call statement('where_pushdown_for_virtual_schema_statement', fetch_result=True) -%}
          select max(crdate) from {{ this }}
    {%- endcall -%}

    {%- set max_date = load_result('where_pushdown_for_virtual_schema_statement')['data'][0][0] -%}

  -- this filter will only be applied on an incremental run
  where "crdate" > '{{ max_date }}'

{% endif %}
```

**Comment by tglunde** [comment]
@ilikutle since you were saying that incremental models are on your TODO list for testing - could you please also consider this?

**Comment by tglunde** [comment]
@djamadeus  fixed in 1.5
